README.md

```diff
@@ -6,7 +6,7 @@ This is Elixir implementation should be compatible with [https://github.com/mgom
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+It is [available in Hex](https://hex.pm/packages/api_auth) and the can be installed
```

Comment:
`and the` what? Either `and can be installed` or `and the package can be installed` is likely what you want here.

Reply:
Good catch. This should be `and can`.